bugsnag-android-core/src/main/java/com/bugsnag/android/Client.java

```diff
@@ -122,7 +124,30 @@ public Client(@NonNull Context androidContext, @NonNull Configuration configurat
         warnIfNotAppContext(androidContext);
         appContext = androidContext.getApplicationContext();
         config = configuration;
-        sessionStore = new SessionStore(config, appContext);
+
+        delegate = new FileStore.Delegate() {
+            @Override
+            public void onErrorIOFailure(Exception exc, File errorFile, String context) {
+                // send an internal error to bugsnag with no cache
+                Thread thread = Thread.currentThread();
+                Error err = new Error.Builder(config, exc, null, thread, true).build();
+                err.setContext(context);
+
+                MetaData metaData = err.getMetaData();
+                metaData.addToTab(INTERNAL_DIAGNOSTICS_TAB, "canRead", errorFile.canRead());
+                metaData.addToTab(INTERNAL_DIAGNOSTICS_TAB, "canWrite", errorFile.canWrite());
+                metaData.addToTab(INTERNAL_DIAGNOSTICS_TAB, "exists", errorFile.exists());
+
+                @SuppressLint("UsableSpace") // requires API 26
```

Comment:
What is the linting issue here? [`getUsableSpace()`](https://developer.android.com/reference/java/io/File.html#getUsableSpace()) seems to available in API 9.

Reply:
The comment was a bit short - I've updated it to say that the alternative API requires API 26.